readme.md

```diff
@@ -12,9 +12,9 @@ The framework essentially spatially applies IPCC national greenhouse gas invento
 It covers only forests converted to non-forests, non-forests converted to forests and forests remaining forests (no other land 
 use transitions). The framework is described and published in [Harris et al. (2021) Nature Climate Change
 "Global maps of twenty-first century forest carbon fluxes"](https://www.nature.com/articles/s41558-020-00976-6).
```

Comment:
"The framework is described and published in [Harris et al. (2021) Nature Climate Change
"Global maps of twenty-first century forest carbon fluxes"](https://www.nature.com/articles/s41558-020-00976-6) and was updated in [Gibbs et al. (2025) "Revised and updated geospatial monitoring of 21st century forest carbon fluxes"](https://essd.copernicus.org/articles/17/1217/2025/).